docs/modules/policies/pages/authoring_tips.adoc

```diff
@@ -2,17 +2,39 @@ include::ROOT:partial$attributes.adoc[]
 
 :policy-schema-file: /policy/v1/Policy.schema.json
 :test-schema-file: /policy/v1/TestSuite.schema.json
+:testdata-principals-schema-file: /policy/v1/TestFixture/Principals.schema.json
+:testdata-resources-schema-file: /policy/v1/TestFixture/Resources.schema.json
+
 :current-policy-schema-url: {app-schema-url-current}{policy-schema-file}
 :latest-policy-schema-url: {app-schema-url-latest}{policy-schema-file}
 :current-test-schema-url: {app-schema-url-current}{test-schema-file}
 :latest-test-schema-url: {app-schema-url-latest}{test-schema-file}
+:current-testdata-principals-schema-url: {app-schema-url-current}{testdata-principals-schema-file}
+:latest-testdata-principals-schema-url: {app-schema-url-latest}{testdata-principals-schema-file}
+:current-testdata-resources-schema-url: {app-schema-url-current}{testdata-resources-schema-file}
+:latest-testdata-resources-schema-url: {app-schema-url-latest}{testdata-resources-schema-file}
+
+
 
 = Policy authoring
 
 == Tips for working with policies
 
 * Policies can be in either YAML or JSON formats. Accepted file extensions are `.yml`, `.yaml` or `.json`. All other extensions are ignored.
-* The JSON schema for Cerbos policies is available at `{current-policy-schema-url}` and the xref:compile.adoc[test suite] schema is available at `{current-test-schema-url}`. If you prefer to always use the latest version, they can be accessed at `{latest-policy-schema-url}` and `{latest-test-schema-url}` respectively.
+* The JSON schema for Cerbos files are avaliable at:
+** Policy
+*** `{current-policy-schema-url}` 
+** xref:compile.adoc[Test suite]
+*** `{current-test-schema-url}`
+** xref:compile.adoc[Principal test fixtures]
+*** `{current-testdata-principals-schema-url}`
+** xref:compile.adoc[Resources test fixtures]
```

Comment:
It'd be nice to link directly to the "Sharing test fixtures" section for these ones I think. Maybe add a `#fixtures` anchor to that section and use it here and below?

Also need to add aux data fixtures too.